jetty-http/src/main/java/org/eclipse/jetty/http/HttpParser.java

```diff
@@ -1271,7 +1271,7 @@ else if (_endOfContent == EndOfContent.UNKNOWN_CONTENT)
                         break;
                     }
                     
-                    if (b==HttpTokens.LINE_FEED && !complianceViolation(RFC2616,"https://tools.ietf.org/html/rfc2616#section-4.2 No colon"))
+                    if (b==HttpTokens.LINE_FEED && !complianceViolation(RFC7230,"https://tools.ietf.org/html/rfc2616#section-4.2 No colon"))
```

Comment:
The `complianceViolation()` call uses RFC7230, but the link points to RFC2616.

Reply:
How did you do this comment?  It only appears here and not in the "Files Changed" tab? 
I was in two minds... RFC2616 actually also enforces No Colon, but we didn't check for it then. Strictly speaking this should be legacy, but I think we should keep behaviour.   Stand by...